include/ppx/geometry.h

```diff
@@ -261,6 +300,10 @@ class Geometry
     void AppendTriangle(const TriMeshVertexData& vtx0, const TriMeshVertexData& vtx1, const TriMeshVertexData& vtx2);
     void AppendEdge(const WireMeshVertexData& vtx0, const WireMeshVertexData& vtx1);
 
+    // Directly overwrites index/vertex buffers
```

Comment:
why do we use pointer here? maybe better with const&? otherwise we need to check if the pointer is nullptr or not